### ISSUE TITLE
I couldn't make the actix example in the book work

### ISSUE THREAD

**Issue by omid** [wafflespeanut/paperclip#222]
This code in the book returns some errors!

https://github.com/wafflespeanut/paperclip/blob/d08e85d4930269befeb81f3bc08149936e915ee6/book/actix-plugin.md

```
use actix_web::{App, HttpServer};
use paperclip::actix::{
    // extension trait for actix_web::App and proc-macro attributes
    OpenApiExt, Apiv2Schema, api_v2_operation,
    // use this instead of actix_web::web
    web::{self, Json},
};
use serde::{Serialize, Deserialize};

// Mark containers (body, query, parameter, etc.) like so...
#[derive(Serialize, Deserialize, Apiv2Schema)]
struct Pet {
    name: String,
    id: Option<i64>,
}

// Mark operations like so...
#[api_v2_operation]
async fn echo_pet(body: Json<Pet>) -> Result<Json<Pet>, ()> {
    Ok(body)
}

#[actix_web::main]
async fn main() -> std::io::Result<()> {
    HttpServer::new(|| App::new()
        // Record services and routes from this line.
        .wrap_api()
        // Add routes like you normally do...
        .service(
            web::resource("/pets")
                .route(web::post().to(echo_pet))
        )
        // Mount the JSON spec at this path.
        .with_json_spec_at("/api/spec")

        // ... or if you wish to build the spec by yourself...

        // .with_raw_json_spec(|app, spec| {
        //     app.route("/api/spec", web::get().to(move || {
        //         actix_web::HttpResponse::Ok().json(&spec)
        //     }))
        // })

        // IMPORTANT: Build the app!
        .build()
    ).bind("127.0.0.1:8080")?
    .run().await
}
```

I'm using the feature "actix", even tested with what we have currently in master.
And I also replaced `api_v2_schema` with `Apiv2Schema`.

The error I can see is:
```
error[E0599]: no method named `wrap_api` found for struct `actix_web::app::App<actix_web::app_service::AppEntry, actix_http::body::Body>` in the current scope
  --> src/main.rs:25:10
   |
25 |         .wrap_api()
   |          ^^^^^^^^ method not found in `actix_web::app::App<actix_web::app_service::AppEntry, actix_http::body::Body>`
```

Anyway, this example is old.

**Comment by wafflespeanut** [comment]
Hi. Which version of actix-web do you have in `Cargo.toml`?

**Comment by omid** [comment]
I have actix-web="3.0"

I also need to change "api_v2_schema" with version 2 of actix

**Comment by wafflespeanut** [comment]
I haven't published support for actix-web 3.0 yet, but it's supported in current `master`. Can you try specifying paperclip as `git` dependency, remove your `Cargo.lock` and try again?

**Comment by omid** [comment]
Thanks, it works. I'll create a tiny PR and fix the "api_v2_schema" and remove actix-rt
